.github/workflows/integrate.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         with:
           provider: microk8s
           channel: 1.29-strict/stable
-          juju-channel: 3.4/stable
+          juju-channel: 3.6/stable
           microk8s-addons: "dns storage rbac metallb:10.64.140.43-10.64.140.49"
 
       # https://github.com/charmed-kubernetes/actions-operator/issues/27
```

Comment:
we need to remove L60 and set the charmcraft channel to `3.x/stable` in the input like the other PRs